src/util/templates/EjsTemplateEngine.ts

```diff
@@ -11,21 +11,27 @@ import Dict = NodeJS.Dict;
  */
 export class EjsTemplateEngine<T extends Dict<any> = Dict<any>> implements TemplateEngine<T> {
   private readonly applyTemplate: Promise<TemplateFunction>;
+  private readonly rootPath: string;
 
   /**
    * @param template - The default template @range {json}
    */
-  public constructor(template?: Template) {
+  public constructor(baseUrl: string, template?: Template) {
     // EJS requires the `filename` parameter to be able to include partial templates
     const filename = getTemplateFilePath(template);
+    this.rootPath = new URL(baseUrl).pathname;
```

Comment:
Would pass the full `baseUrl` to the template instead of the sliced off path. Either one works for the HTML and using the full URL is a bit more consistent with how we use it everywhere else.